lib/storage/metadata/MetadataWrapper.js

```diff
@@ -226,6 +226,20 @@ class MetadataWrapper {
         });
     }
 
+    getBucketQuota(bucketName, log, cb) {
+        log.debug('getting bucket quota from metadata');
+        this.client.getBucketAttributes(bucketName, log, (err, data) => {
+            if (err) {
+                log.debug('error from metadata', { implName: this.implName,
+                    error: err });
+                return cb(err);
+            }
+            log.trace('bucket retrieved from metadata');
```

Comment:
We can remove the trace log as we already have a debug one, and another one in case of error. This won't give much help as in case of error, we'll have the debug ones already